leaders/src/main/java/edu/cornell/Main.java

```diff
@@ -0,0 +1,43 @@
+package edu.cornell;
+
+import edu.cornell.testconsumer.PrintTestConsumer;
+import edu.cornell.testconsumer.TestConsumer;
+import java.util.List;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * The main leader application
+ * Environment variables:
+ * SPEED_REPO_URL: the url of the repository to clone
```

Comment:
I would make these static constants in a separate file

Reply:
When we spin up a new leader (because each pipeline will use one leader for coordination, so multiple leaders), we need to pass these inputs into that leader. It might be difficult to have these in a config file since the leader won't know where that config file is. I think env variables are easier because they can be passed into the Docker container on startup